apps/taxes/package.json

```diff
@@ -83,6 +83,6 @@
   },
   "private": true,
   "saleor": {
-    "schemaVersion": "3.10"
+    "schemaVersion": "3.15"
```

Comment:
please update docs and minimum saleor version to according schema

Reply:
Saleor added order fully refunded in 3.14, does it need to be that version for something else?